src/app/pages/BuyPage/components/AmountInput/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { bignumber } from 'mathjs';
+import React, { useMemo } from 'react';
+import cn from 'classnames';
+
+import { Asset } from '../../../../../types';
```

Comment:
It seems better to use absolute path instead of relative path.
`import { Asset } from 'types';`

Reply:
fixed